<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoanManagerDialog</class>
 <widget class="QDialog" name="LoanManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Préstamos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gestión de Préstamos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_loans">
      <attribute name="title">
       <string>Préstamos de Libros</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_filter">
           <property name="text">
            <string>Filtrar por:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmb_loan_filter">
           <item>
            <property name="text">
             <string>Todos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Activos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vencidos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Devueltos</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_loans">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>ID Préstamo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID Usuario</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID Libro</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fecha Inicio</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fecha Vencimiento</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Estado</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>ID Usuario:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="txt_user_id_loan">
             <property name="placeholderText">
              <string>ID del usuario</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>ID Libro:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txt_book_id_loan">
             <property name="placeholderText">
              <string>ID del libro</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Fecha Inicio:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDateEdit" name="date_start"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Fecha Vencimiento:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDateEdit" name="date_end"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Estado:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="cmb_status_loan">
             <item>
              <property name="text">
               <string>Activo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vencido</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Devuelto</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txt_loan_id">
             <property name="placeholderText">
              <string>ID del préstamo</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>ID Préstamo:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_search_loan">
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_add_loan">
             <property name="text">
              <string>Agregar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_update_loan">
             <property name="text">
              <string>Actualizar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_delete_loan">
             <property name="text">
              <string>Eliminar</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_laptops">
      <attribute name="title">
       <string>Préstamos de Laptops</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_filter_laptop">
           <property name="text">
            <string>Filtrar por:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmb_laptop_filter">
           <item>
            <property name="text">
             <string>Todos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Activos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vencidos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Devueltos</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_laptops">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <column>
          <property name="text">
           <string>ID Préstamo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID Usuario</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID Laptop</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fecha Inicio</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fecha Vencimiento</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Estado</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>ID Usuario:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="txt_user_id_laptop">
             <property name="placeholderText">
              <string>ID del usuario</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>ID Laptop:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txt_laptop_id">
             <property name="placeholderText">
              <string>ID de la laptop</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Fecha Inicio:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDateEdit" name="date_start_laptop"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Fecha Vencimiento:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QDateEdit" name="date_end_laptop"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Estado:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="cmb_status_laptop">
             <item>
              <property name="text">
               <string>Activo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vencido</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Devuelto</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txt_laptop_loan_id">
             <property name="placeholderText">
              <string>ID del préstamo</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>ID Préstamo:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="btn_search_laptop">
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_add_laptop">
             <property name="text">
              <string>Agregar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_update_laptop">
             <property name="text">
              <string>Actualizar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_delete_laptop">
             <property name="text">
              <string>Eliminar</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoanManagerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoanManagerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
